ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -74,16 +83,21 @@ sub handle_config_file
             my $ref = ref($val);
             if($ref) {
                 if($ref eq 'ARRAY') {
+                    if ($option_set eq 'multiline_options') {
+                        $fh->remove_lines(qr{(?i)^\W*$option(?:\s+\S+)+}, qr{^#});
```

Comment:
The other nice thing about this is that it provides a nice upgrade path for us to update the templates before the code is upgraded.

We can leave stuff in options (temporarily) to keep compatability with the old code, then if something goes into multiline_options it removes what was set in options.